.gitignore

```diff
@@ -55,3 +55,5 @@ nbactions.xml
 .idea/
 *.iml
 *.iws
+gateway.properties
```

Comment:
Please remove those two entries inside gitignore since we may use this in future